src/taco-cli/cli/utils/cordovaWrapper.ts

```diff
@@ -151,7 +151,7 @@ class CordovaWrapper {
         CordovaHelper.prepareCordovaConfig(cordovaParameters);
         return CordovaHelper.wrapCordovaInvocation<any>(cordovaCliVersion, (cordova: Cordova.ICordova) => {
             return cordova.raw.create(cordovaParameters.projectPath, cordovaParameters.appId, cordovaParameters.appName, cordovaParameters.cordovaConfig);
-        }, tacoUtility.InstallLogLevel.taco);
```

Comment:
Wasn't this log level added specifically for the create command, so we would print messages "Installing cordova X" and "Successfully installed cordova X"?